pkg/bgpv1/agent/controller.go

```diff
@@ -31,7 +32,7 @@ import (
 
 var (
 	log                   = logging.DefaultLogger.WithField(logfields.LogSubsys, "bgp-control-plane")
-	defaultInformerResync = 30 * time.Second
+	defaultInformerResync = 5 * time.Minute
```

Comment:
let's remove informer resync

Reply:
This is OK? There are no possibility of missed events from the watchers?

Reply:
we will always receive an event with the latest state of a resource. If an event is missed the `defaultInformerResync = 5 * time.Minute` won't help because that doesn't refetch the state from kube-apiserver.

Reply:
Ah good point. Okay I'll nix.